### ISSUE TITLE
Html documentation generation fails

### ISSUE THREAD

**Issue by EdwardDiehl** [KlausBenndorf/guide4you#278]
Hi, the library looks pretty nice and i'd like to use this library in one of my pet-projects.

I tried to generate html documentation with the script `doc` from the `package.json` file. I installed `esdoc-standard-plugin` plugin and added it to the esdoc.json config file into the `plugins` section.
The script fails with this output:
```
...
resolve: markdown in description
resolve: test relation
output: /home/edward/work/projects/my/guide4you/doc/identifiers.html
output: /home/edward/work/projects/my/guide4you/doc/index.html
output: /home/edward/work/projects/my/guide4you/doc/class/src/controls/ActivatableMixin.js~ActivatableMixin.html
output: /home/edward/work/projects/my/guide4you/doc/class/src/controls/ActiveGroup.js~ActiveGroup.html
output: /home/edward/work/projects/my/guide4you/doc/class/src/controls/ArrowButtons.js~ArrowButtons.html

SyntaxError: Invalid regular expression: /[~][string$/: Unterminated character class
    at new RegExp (<anonymous>)
    at ClassDocBuilder._findByName (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:116:20)
    at ClassDocBuilder._buildDocLinkHTML (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:768:22)
    at ClassDocBuilder._buildTypeDocLinkHTML (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:750:19)
    at v.split.map.vv (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:729:23)
    at Array.map (<anonymous>)
    at inner.split.map.v (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:727:29)
    at Array.map (<anonymous>)
    at ClassDocBuilder._buildTypeDocLinkHTML (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:726:43)
    at ClassDocBuilder._buildSignatureHTML (/home/edward/work/projects/my/guide4you/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:854:34)
...
```

Maybe you know how to fix this issue?

Another one related question - does the library has some online documentation? As i see there is no online documentation and the only way to get it is to generate it with this script.

Thank you.

**Comment by simonseyock** [comment]
Thanks for reporting that issue. I will look into it.
Uploading the documentation is a good idea, I think I will make it available at the github.io page.
Also I want to notice that guide4you is not that much of a library but more of an configurable and extensible program.

**Comment by simonseyock** [comment]
Fixed by #280 
See https://klausbenndorf.github.io/guide4you/index.html
